lib/timeline/Timeline.js

```diff
@@ -57,12 +57,17 @@ function Timeline (container, items, groups, options) {
     minHeight: null
   };
   this.options = util.deepExtend({}, this.defaultOptions);
-  if (options) {
-    this.options.rtl = options.rtl
-  }
+
+
   // Create the DOM, props, and emitter
   this._create(container);
 
+  if (!options || (options && typeof options.rtl == "undefined")) {
+    this.options.rtl = window.getComputedStyle(this.dom.root, null).direction == "rtl";
```

Comment:
You could just change it and push the changes to your branch. Would be cool, if you test it before, I didn't ;-)